robottelo/cli/hammer.py

```diff
@@ -38,11 +40,37 @@ def _csv_reader(output):
             yield row
 
 
+def _normalize_header(header):
+    """Replace empty spaces with '-' and lower all chars"""
+    return header.replace(' ', '-').lower()
+
+
+def parse_json(stdout):
+    """Parse JSON output from Hammer CLI and convert it to python dictionary
+    while normalizing keys."""
+    parsed = json.loads(stdout)
+    return _normalize_obj(parsed)
+
+
+def _normalize_obj(obj):
+    """Normalize all dict's keys replacing empty spaces with "-" and lowering
+    chars"""
+    if isinstance(obj, dict):
+        return {_normalize_header(k): _normalize_obj(v) for k, v in
+            obj.items()}
+    elif isinstance(obj, list):
+        return [_normalize_obj(v) for v in obj]
+    # doing this to conform to csv parser
+    elif isinstance(obj, int) and not isinstance(obj, bool):
+        return str(obj)
```

Comment:
@elyezer once @oshtaier is has another issue blocked by this one, I thought would be better for know conform with current csv parser to not impact in so many places and maybe creating another issue to clean this up. What you think?